core/server/api/v2/db.js

```diff
@@ -5,6 +5,57 @@ const importer = require('../../data/importer');
 const common = require('../../lib/common');
 const models = require('../../models');
 
+class AsyncImporter {
```

Comment:
I'm not entirely sure where to put this; it definitely doesn't belong here!

Reply:
I see 2 options:

- either make `AsyncImporter` generic as possible (no dependency to the importer) and make it re-useable - a polling class
- or just put it straight to the importer for now, because it has a depdendency to the importer e.g. `importer.importFromFile({polling:true})` returns an instance of the polling class

Reply:
I'm thinking of implementing an abstract polling class - where do you suggest I put the exported class file?

Reply:
If it's super generic then I would put it to lib/?

Reply:
Alright sounds good! I'll try to push tonight or this weekend 😄